app/screens/Account/profile.js

```diff
@@ -38,20 +66,50 @@ class ProfileScreen extends Component {
     return (
       <Wrapper headerView={headerView}>
```

Comment:
To get the bottom button position correctly: 

It might work to move the `baseStyles.wrapperContent` object to the `Wrapper` component.

And then remove that view that currently has the `baseStyles.wrapperContent` object.

That's what I'm guessing based on looking at the surveys screen: https://github.com/osmlab/field-data-collection/blob/master/app/screens/Account/surveys.js